### PR TITLE
fix item_appeared

### PR DESCRIPTION
**Pull request by ya4ept**
nameID changed from 2 bytes to 4 bytes

should fix this: https://github.com/OpenKore/openkore/commit/10b225b64508ab0a7771fa4b37bc048d6ba703ab#commitcomment-47254947

### CODE REVIEW

File: src/Network/Receive/Zero.pm [hunk 1]
```diff
@@ -24,6 +24,13 @@ sub new {
 	my ($class) = @_;
 	my $self = $class->SUPER::new(@_);
 
+	my %packets = (
+		'0ADD' => ['item_appeared', 'a4 V v C v2 C2 v C v', [qw(ID nameID type identified x y subx suby amount show_effect effect_type)]],
+	);
+
+	$self->{packet_list}{$_} = $packets{$_} for keys %packets;
+=pod
+# I think this is extra code in the received packages
```
Comment (alisonrag) [inline src/Network/Receive/Zero.pm:33]: this is used in xkore 3 and 2, if not set, openkore will use old character pages and other old packets, and cRo has support for both cause use nprotect gameguard

**Review by alisonrag — CHANGES_REQUESTED** [review]

**Review by alisonrag — APPROVED** [review]

**Comment by alisonrag** [comment]
just wait the @hubertyang88 test

**Review by alisonrag — CHANGES_REQUESTED** [review]
sorry i just saw the file name now, cRO is at cRO.pm, this server is not equals to Zero.pm
so the changes should be done at Receive/cRO.pm

**Comment by hubertyang88** [comment]
================================================
<< Received packet:      0ADD - Item Exists [24 bytes]   Feb 19 18:47:02 2021
  0>  DD 0A 8A 85 00 00 FC 01    00 00 00 00 01 9F 00 BB    ................
 16>  00 06 0C 02 00 00 00 00                               ........
Item appeared : name (0) x 2 (159, 187)

I add '0ADD' to cRo.pm , it works !
I ignored other "packets changes" in zRo.pm , because my op works fine , should I change the file like  zRo.pm?
[cRO.txt](https://github.com/OpenKore/openkore/files/6009425/cRO.txt)
this is my cro.pm

**Comment by ya4ept** [comment]
moved the changes to cro.pm
@hubertyang88 please test again

PS; did the hard discard all changes at zero.pm

**Review by alisonrag — APPROVED** [review]

**Comment by hubertyang88** [comment]
I replace cro.pm with new one , it looks good , no problem found , Thanks!

**Pull request merged**